decisionengine/decisionengine_endpoint/recieve_rock_info.py

```diff
@@ -40,13 +40,31 @@ def find_path():
         if request.get_header(head) != headers.get(head):
             abort(406, 'Incorrect headers')
     rocks = request.json.get('rocks')
-    print(rocks)
     if not rocks:
         abort(400, 'Bad request')
     for rock in rocks:
         if not (rock.get('placement') or rock.get('type')):
             abort(400, 'Bad request')
 
+    userHeightRequest = request.json.get('userHeight')
+    if userHeightRequest is None:
+        userHeightRequest = '' 
+    elif not isinstance(userHeightRequest, (int, float, complex)):
+        abort(400, 'Bad user height number input')
+
+    wallHeightRequest = request.json.get('wallHeight')
+    if wallHeightRequest is None:
+        wallHeightRequest = '' 
+    elif not isinstance(wallHeightRequest, (int, float, complex)):
+        abort(400, 'Bad wall height number input')
+
+    endPts = {
+        "rocks":rocks,
+        "userHeight":userHeightRequest,
+        "wallHeight":wallHeightRequest
```

Comment:
I would prefer to not get key-value pairs back that the user did not submit, even if the are empty strings. The client sending the request should only see userHeight and wallHeight if they sent in the values for them. Would you instead only add them to the dictionary if they are in the request?

Also space on the right side of the :

Reply:
fixed everything, plz check!